### PR TITLE
add basic .gitignore file

### PR DESCRIPTION
**Pull request by nitishbharti235**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,27 +2,49 @@
...
+            I like watching anime.
+            I like solving DSA problems. I just like writing codes and seeing that thing working.
+            and I do things deliberately and honestly with my heart and my energy.
+            Focus and Patient these two key things I follow.
+          </p>
+            <ul>
+              <li><a href="https://www.linkedin.com/in/nitish-bharti/">LinkedIn Profile</a></li>
+              <li><a href="https://github.com/nitishbharti235">Github Profile</a></li>
+            </ul>
+        </p>
```
Comment (pitankar) [inline portfolio/src/main/webapp/index.html:28]: Is this an extra `</p>` or the one on line 17 is?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,77 @@
...
+  line-height: 1.6em;
+  margin: 0;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.container{
+  width: 80%;
+    margin: auto;
+    overflow: hidden;
```
Comment (pitankar) [inline portfolio/src/main/webapp/style.css:13]: The indentation seems different compared to elsewhere in file.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/index.html:28]: ahh.. this on line 28 is an extra </p> tag.. removing it

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/style.css:13]: fixing it

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,27 +2,48 @@
...
+
+      <div class="container">
+        <h2>This is my portfolio page and here is my faviourite anime character Ken kaneki from the series Tokyo Ghoul.</h2>
+        <img src="images/kaneki.png" alt="Ken Kaneki Picture.png"/>
+      </div>
+
+      <div class="container">
+        <p id="p-click">Click here to get a random Tokyo Ghoul's Dark Quote:</p>
+      </div>
+      <div class="container">
```
Comment (pitankar) [inline portfolio/src/main/webapp/index.html:39]: nit: Considering the format elsewhere in the file. the `<div>` seem to be starting after an empty line. same should be followed here? newline on 39?

**Pull request merged**